docs/api.yml

```diff
@@ -0,0 +1,226 @@
+    post:
+      summary: Cancel the job with the specified job_id
+      parameters:
+        - name: job_id
+          in: path
+          schema:
+            type: string
+            format: uuid
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
+      summary: Delete the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queues.json:
+    get:
+      summary: List Queues and their state
+      responses:
+        200:
+          description: |
+            Return object containting list of queues and theis states
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  queues:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                        count:
+                          type: integer
+                        url:
+                          type: string
+                          format: url
+  /jobs/{queue_name}/{page}.json:
+    get:
+      summary: List workers and their state
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+        - name: page
+          in: path
+          schema:
+            type: integer
+          required: true
+      responses:
+        200:
+          description: |
+            Return object containting list of worker states in workers property
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  jobs:
+                    type: array
```

Comment:
I think we should mention sorting of returned jobs here

Reply:
```suggestion
                    type: array
                    description: |
                      Default ordering is oldest entries first, except for the
                      `[failed]` and `[finished]` registries, where it is newest first.
                    
```